docs/e2e/README.md

```diff
@@ -0,0 +1,81 @@
+#### Running e2e tests locally with go test
+
+If you are developing tests locally, you could use the native `go test` to invoke your ginkgo test specs.
+
+Example: Run the end user e2e tests
+```
+export FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]\[Fake\]"
+export TIMEOUT=20m
+./hack/e2e.sh
+```
+
+As a shorthand to the above, you could simply run `make e2e`
+
+Example: Run the scale up/down e2e tests against the fake rp
+```
+source ./env
+export FOCUS="\[ScaleUpDown\]\[Fake\]"
+export TIMEOUT=30m
+export EXTRA_FLAGS="-scaleUpManifest=test/manifests/normal/scaleup.yaml -scaleDownManifest=test/manifests/normal/scaledown.yaml"
+./hack/e2e.sh
+```
+
+#### Running e2e tests with the container image
+
+To run the e2e tests in a container use the [e2e](https://quay.io/repository/openshift-on-azure/e2e-tests) container 
+image
+
+```
+docker pull quay.io/openshift-on-azure/e2e-tests:latest
+```
+
+Example: Run the end user e2e tests in a container
+```
+export FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]\[Fake\]"
+export TIMEOUT=20m
+
+docker run \
+--rm \
+-it \
+--volume $PWD/_data:/_data \
+--volume $PWD/artifacts:/artifacts \
```

Comment:
Haven't used this `artifacts` directory at the root of `openshift-azure` before. May be worth adding in `.gitignore`